blackbox/docs/release_notes/1.2.0.txt

```diff
@@ -0,0 +1,157 @@
+.. _version_1.2.0:
+
+=============
+Version 1.2.0
+=============
+
+Released on 2017/04/18.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.1.2` or higher before you upgrade to 1.2.0.
+
+   You cannot perform a :ref:`cluster_upgrade` to this version. Any upgrade to
+   this version will require a full cluster restart.
+
+.. WARNING::
+
+   Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/a/backing-up-and-restoring-crate/
+
+Breaking Changes
+================
+
+ - Due to the support of user defined functions there are new reserved keywords
+   in the Crate SQL dialect: ``RETURNS``, ``CALLED`, ``REPLACE``, ``FUNCTION``,
+   ``LANGUAGE``, ``INPUT``
+
+ - The ``enterprise.license`` setting is set to ``true`` by default.
+   If you start crate enterprise without a valid enterprise license,
```

Comment:
"crate" -> "CrateDB"

this sentence doesn't make sense to me. is it saying that you start "crate enterprise" by having the "enterprise.license" set to true? if so that should be made explicit

but then it says if I do this, and I don't have a valid license (which it doesn't tell me how I would know that or how crate would know that) then I am in "unlicensed mode". but that conflicts with the idea that I have "license" set to "true"

this sentence needs making more clear